cookbooks/bcpc/templates/default/apache-mirror.erb

```diff
@@ -83,4 +83,13 @@
             allow from all
         </IfModule>
     </Directory>
+
+   Alias "/cluster-def/" "/home/vagrant/cluster/cluster.txt"
```

Comment:
Don't we have a node attribute for the user? OpenStack and test-kitchen clusters did not use a `vagrant` user.